events.res

```diff
@@ -1,721 +0,0 @@
-# This file contains a list of all events in ENIGMA and the order in which they are executed.

```

Comment:
There are 2 more copies of this file in the codebase. ones in / and is something like localeventres and the other is next to the event_reader code